src/base58.cpp

```diff
@@ -225,7 +225,7 @@ class CBitcoinAddressVisitor : public boost::static_visitor<bool>
     bool operator()(const CNoDestination& no) const { return false; }
 };
 
-} // anon namespace
+} // namespace
```

Comment:
@jtimon My reasoning behind that was to achieve consistency how anonymous namespaces are indicated. Consistency in order to improve readability and allow for automatic identification of missing/incorrect end of namespace comments.

`// namespace` is the form preferred by the clang-tooling (`clang-tidy`, etc).

Let me know if the choice of `// anon namespace` is intentional and I'll adjust this PR :-)